.github/workflows/ci.yml

```diff
@@ -8,8 +8,16 @@ on:
     branches: [ master ]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container: urnai/linter:latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: lint-commit
```

Comment:
`lint-commit` needs a target branch as a parameter. The correct value is `origin/master`.

Reply:
understood